Packages/cdms2/Lib/axis.py

```diff
@@ -735,10 +735,22 @@ def designateTime(self, persistent=0, calendar=None):
             if calendar is not None:
                 self.setCalendar(calendar, persistent)
 
+    # For isTime(), keep track of whether each id is for a time axis or not, for better performance.
+    # This dictionary is a class variable (not a member of any particular instance).
+    idtaxis = {}  # id:type where type is 'T' for time, 'O' for other
+
```

Comment:
Yes, it would break!  I've never seen such a thing, but among the changes I've made, this is the one which I was least sure about.